internal/service/dns_config/api_forward_zone_resource_test.go

```diff
@@ -0,0 +1,553 @@
+package dns_config_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+
+	"github.com/infobloxopen/bloxone-go-client/dns_config"
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/acctest"
+)
+
+//TODO: add tests
+// The following require additional resource/data source objects to be supported.
+// - internal_forwarders
+// - hosts
+
+func TestAccForwardZoneResource_basic(t *testing.T) {
+	var resourceName = "bloxone_dns_forward_zone.test"
+	var v dns_config.ConfigForwardZone
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccForwardZoneBasicConfig("tf-acc-test.com."),
```

Comment:
can you use a random name for all these tests?

Reply:
Resolved